internal/common/objects/MegaBike.go

```diff
@@ -12,12 +12,15 @@ type IMegaBike interface {
 	RemoveAgent(bikerId uuid.UUID)
 	GetAgents() []IBaseBiker
 	UpdateMass()
+	KickOutAgent(bikerId uuid.UUID)
```

Comment:
here the biker id seems to be an argument whereas in the function definition is what's returned by the function. this is causing the build to fail so it needs to be fixed